app/src/main/java/com/mashup/dorabangs/navigation/MainNavHost.kt

```diff
@@ -81,30 +83,28 @@ fun MainNavHost(
             onClickBackIcon = { folderType ->
                 val isVisibleBottomSheet = folderType == FolderManageType.CREATE
                 appState.navController.previousBackStackEntry?.savedStateHandle?.set("isVisibleBottomSheet", isVisibleBottomSheet)
-                appState.navController.previousBackStackEntry?.savedStateHandle?.set("editFolderName", "")
+                appState.navController.previousBackStackEntry?.savedStateHandle?.set("isChange", false)
```

Comment:
```suggestion
                appState.navController.previousBackStackEntry?.savedStateHandle?.set("isChanged", false)
```
isChange보단 isChanged가 나아 보이네용

Reply:
좋습니당